runners/spark/src/main/java/org/apache/beam/runners/spark/SparkRunner.java

```diff
@@ -150,7 +150,7 @@ public EvaluationResult run(Pipeline pipeline) {
 
         return ctxt;
       }
-    } catch (Exception e) {
+    } catch (Throwable e) {
```

Comment:
I think findbugs is misinterpreting this as it (rightfully) expects only a `RuntimeException`.
I actually wanted to catch `Throwable` instead anyway here so if this satisfies findbugs as well, cheers!